examples/urlInterpolationDemo.html

```diff
@@ -42,21 +42,21 @@ <h2>Localization Demo</h2>
                 var DEFAULT_LANG_TAG = 'en-us';
                 var langTag = null;
                 var welcomeTokens = {
```

Comment:
Not sure if these were localized by the localization team, but figured this change was still reasonable.

Reply:
Yeah, I think I picked these up from a bundle somewhere just to illustrate.  Should be good to change here.